app/config/env/platformsh.php

```diff
@@ -132,3 +134,28 @@
         $container->setParameter('solr_core', substr($endpoint['path'], 5));
     }
 }
+
+// We will pick a varnish route by the following prioritization:
+// - The first route found that has upstream: varnish
+// - if primary route has upstream: varnish, that route will be prioritised
+// If no route is found with upstream: varnish, then purge_server will not be set
+$route = null;
+foreach ($routes as $host => $info) {
+    if ($route === null && $info['type'] === 'upstream' && $info['upstream'] === 'varnish') {
+        $route = $host;
+    }
+    if ($info['type'] === 'upstream' && $info['upstream'] === 'varnish' && $info['primary'] === true) {
+        $route = $host;
+        break;
+    }
+}
+
+if ($route !== null) {
+    $container->setParameter('purge_type', 'varnish');
+    $container->setParameter('purge_server', rtrim($route, '/'));
+}
+
+// Setting default value for HTTPCACHE_VARNISH_INVALIDATE_TOKEN if it is not explicitly set
+if (!$varnishInvalidateToken = getenv('HTTPCACHE_VARNISH_INVALIDATE_TOKEN')) {
```

Comment:
```suggestion
if (!getenv('HTTPCACHE_VARNISH_INVALIDATE_TOKEN')) {
```
No point in assigning to a variable you're not going to use, if I read this correctly.